packages/backend/src/open_payments/payment/outgoing/routes.ts

```diff
@@ -128,9 +128,9 @@ function outgoingPaymentToBody(
   return Object.fromEntries(
     Object.entries({
       ...outgoingPayment.toJSON(),
-      // paymentPointer: `${deps.config.publicHost}/${outgoingPayment.paymentPointerId}`,
-      accountId: `${deps.config.publicHost}/${outgoingPayment.paymentPointerId}`,
-      id: `${deps.config.publicHost}/${outgoingPayment.paymentPointerId}/outgoing-payments/${outgoingPayment.id}`,
+      id: `${outgoingPayment.paymentPointer.url}/outgoing-payments/${outgoingPayment.id}`,
+      paymentPointer: outgoingPayment.paymentPointer.url,
+      paymentPointerId: undefined,
```

Comment:
Should we be returning the this? I don't think so

Reply:
as in remove `paymentPointerId: undefined,`?
which should be fine as long as the `toJSON` gets updated to exclude `paymentPointerId`, which should also be fine

Reply:
I was able to change this for outgoing payments, but `toJSON` is being called for the request to `quoteUrl` which expects `paymentPointerId`:
https://github.com/interledger/rafiki/blob/main/packages/backend/src/open_payments/quote/service.ts#L289